content/en/docs/reference/networking/service-protocols.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Protocols for Services
+content_type: reference
+---
+
+<!-- overview -->
+If you configure a {{< glossary_tooltip text="Service" term_id="service" >}},
+you can select from any network protocol that Kubernetes supports.
+
+Kubernetes supports the following protocols with Services:
+
+- [`SCTP`](#protocol-sctp)
+- [`TCP`](#protocol-tcp) _(the default)_
+- [`UDP`](#protocol-udp)
+
+When you define a Service, you can also specify the
+[application protocol](/docs/concepts/services-networking/service/#application-protocol)
+that it uses.
+
+This document details some special cases, all of them typically using TCP
+as a transport protocol:
+
+- [HTTP](#protocol-http-special) and [HTTPS](#protocol-http-special)
+- [PROXY protocol](#protocol-proxy-special)
+- [TLS](#protocol-tls-special) termination at the load balancer
+
+<!-- body -->
+## Supported protocols {#protocol-support}
+
+There are 3 valid values for the `protocol` of a port for a Service:
+
+### `SCTP` {#protocol-sctp}
+
+{{< feature-state for_k8s_version="v1.20" state="stable" >}}
+
+When using a network plugin that supports SCTP traffic, you can use SCTP for
+most Services. For `type: LoadBalancer` Services, SCTP support depends on the cloud
+provider offering this facility. (Most do not).
+
+SCTP is not supported on nodes that run Windows.
+
+#### Support for multihomed SCTP associations {#caveat-sctp-multihomed}
+
+The support of multihomed SCTP associations requires that the CNI plugin can support the assignment of multiple interfaces and IP addresses to a Pod.
+
+NAT for multihomed SCTP associations requires special logic in the corresponding kernel modules.
+
+{{< note >}}
+The kube-proxy does not support the management of SCTP associations when it is in userspace mode.
+{{< /note >}}
+
+
+### `TCP` {#protocol-tcp}
+
+You can use TCP for any kind of Service, and it's the default network protocol.
+
+### `UDP` {#protocol-udp}
+
+You can use UDP for most Services. For `type: LoadBalancer` Services,
+UDP support depends on the cloud provider offering this facility.
+
+
+## Special cases
+
+### HTTP {#protocol-http-special}
+
+If your cloud provider supports it, you can use a Service set to `type: LoadBalancer` as a way
+to set up external HTTP / HTTPS reverse proxying, forwarded to the EndpointSlices / Endpoints of that Service.
+
+Typically, you set the protocol to `TCP` and add an
+{{< glossary_tooltip text="annotation" term_id="annotation" >}}
+(usually specific to your cloud provider) that configures the load balancer
+to handle traffic at the HTTP level.
+This configuration might also include serving HTTPS (HTTP over TLS) and
+reverse-proxying plain HTTP to your workload.
+
+{{< note >}}
+You can also use an {{< glossary_tooltip term_id="ingress" >}} to expose
+HTTP/HTTPS Services.
+{{< /note >}}
+
+You might additionally want to specify that the
+[application protocol](/docs/concepts/services-networking/service/#application-protocol)
```

Comment:
I can't find this reference

Reply:
Does https://deploy-preview-30817--kubernetes-io-main-staging.netlify.app/docs/concepts/services-networking/service/#application-protocol work?